src/core/storage-services/Holochain/HolochainService.ts

```diff
@@ -123,6 +134,37 @@ export default class HolochainService {
                 this.#appWebsocket = await AppWebsocket.connect(`ws://localhost:${this.#appPort}`, 100000, this.handleCallback.bind(this))
                 console.debug("HolochainService: Holochain app interface connected on port", this.#appPort)
             };
+
+            //Install signing service DNA
+            const activeApps = await this.#adminWebsocket!.listApps({status_filter: AppStatusFilter.Enabled});
+            if (!activeApps.map(value => value.installed_app_id).includes("signing_service")) {
+                const pubKey = await this.pubKeyForLanguage("main");
+
+                const hash = await this.#adminWebsocket!.registerDna({
+                    //Pretty sure this is not gonna work in production
```

Comment:
Let's just publish the DNA as release in its repo and download it the same way we download the bootstrap languages..

Reply:
hah yeah right...